recipes/rayoptics/meta.yaml

```
@@ -0,0 +1,67 @@
+{% set name = "rayoptics" %}
+{% set version = "0.6.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b04308201087a3dd6ba2083e3ef58cc852ea597bb0e5d3545c8b6bdb074ffb7c
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - rayoptics = rayoptics.qtgui.rayopticsapp:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pyscaffold >=3.1a0,<3.2a0
+    - python >=3.7
+  run:
+    - attrs >=19.1.0
+    - ipywidgets >=7.5.1
+    - json_tricks >=3.13.2
+    - matplotlib >=3.1.1
+    - numpy >=1.17.1
+    - opticalglass >=0.6.5
+    - pandas >=0.25.1
+    - pyqt <5.13
+    - python >=3.7
+    - qdarkstyle >=2.8
+    - qtconsole >=4.5.5
+    - requests >=2.23.0
+    - scipy >=1.3.1
+    - traitlets >=4.3.2
+    - transforms3d >=0.3.1
+
+test:
+  imports:
+    - rayoptics
```

Is there a reason for not including pip check here?
```suggestion
    - rayoptics
  commands:
    - pip check
  requires:
    - pip
```

---

I think by removing pip check, I removed the symptom of a problem, but not the problem. With pip check, it flags a dependency of qdarkstyle, helpdev, as missing. This package is on pipy and hosted in GitLab but not on conda-forge or in anaconda.
I presume the correct solution would be for me to set up helpdev as a conda-forge submission. If there's another approach, please let me know.
Thank you for your help.

---

OK, I'm stuck now. I tried including helpdev and there's a message about conda-verify failing that I think prevents the build from completing. I tried reproducing on my local machine and conda-verify seems happy.
I admit I'm a little mystified that qdarkstyle is available on conda-forge but appears to depend on helpdev which isn't on conda-forge.

---

I've examined the use of helpdev by the qdarkstyle package. It is only used if you run the qdarkstyle package as a script to document the state of your environment, e.g. for a bug report. It is not used if you are using qdarkstyle programmically with pyqt, as is done with rayoptics (and probably all other packages depending on qdarkstyle).
I am going to remove the pip check test and helpdev from this PR.

---

Indeed, this is a bug in the `qdarkstyle` recipe. I've made a PR to fix it and the "pip check" test can be re-added after https://github.com/conda-forge/qdarkstyle-feedstock/pull/14 is merged.

That said, there is no reason for it to hold up this PR so LGTM 👍